### PR TITLE
Update api_key to docs-demo

### PR DESCRIPTION
**Pull request by opman129**

### CODE REVIEW

File: nft/nfts.yaml [hunk 1]
```diff
@@ -149,7 +148,6 @@ paths:
               properties:
                 tokens:
                   type: array
-                  description: list of token objects to batch request NFT metadata for
```
Comment (elanh) [inline nft/nfts.yaml:152]: @opman129 why did we remove this?

File: nft/nfts.yaml [hunk 2]
```diff
@@ -399,7 +397,6 @@ paths:
                 properties:
                   owners:
                     $ref: '#/components/schemas/owners'
-      operationId: getOwnersForToken
```
Comment (elanh) [inline nft/nfts.yaml:402]: same with this?

File: nft/nfts.yaml [hunk 3]
```diff
@@ -533,32 +530,6 @@ paths:
...
-          - go
-      responses:
-        '200':
-          description: ''
-          content:
-            application/json:
-              schema:
-                type: boolean
-                description: <br/> True - if the queried token is marked as an airdrop. <br/> False - if the queried token is not marked as an airdrop.
-      operationId: isAirdrop
```
Comment (elanh) [inline nft/nfts.yaml:561]: wondering if this was actually a merge conflict that happened? do u know what's going on here?

File: nft/nfts.yaml [hunk 4]
```diff
@@ -854,16 +826,24 @@ paths:
...
+          required: true
         - $ref: '#/components/schemas/toBlock'
+          required: true
         - $ref: '#/components/schemas/order'
+          required: true
         - $ref: '#/components/schemas/marketplace'
+          required: true
         - $ref: '#/components/schemas/contractAddress'
           description: String - The contract address of a NFT collection to filter sales by. Defaults to returning all NFT contracts.
+          required: true
```
Comment (elanh) [inline nft/nfts.yaml:840]: i don’t think these are all required

Comment (elanh) [reply nft/nfts.yaml:840]: but can u double check?

**Review by elanh — CHANGES_REQUESTED** [review]
@opman129 can u re-pull from master and check the changes again, looks like there's a bunch of stuff in here that's not meant to be

File: nft/nfts.yaml (hunk 4 above)
Comment (opman129) [reply nft/nfts.yaml:840]: I'll just revert with a new PR and cleared all conflicts.

**Pull request closed without merging**